dynamodb_3.tf

```diff
@@ -13,7 +13,7 @@ resource "null_resource" "dynamodb3_checker" {
 
 module "dynamodb3" {
```

Comment:
Funny how I have absolutely zero clue what the heck dynamodb_2 or dynamodb_3 means 😄